package.json

```diff
@@ -21,19 +21,13 @@
     "@auto-it/all-contributors": "^10.37.6",
     "@auto-it/first-time-contributor": "^10.37.6",
     "@auto-it/slack": "^10.37.6",
-    "@grafana/eslint-config": "^2.5.0",
+    "@grafana/eslint-config": "^5.1.0",
     "@grafana/tsconfig": "^1.2.0-rc1",
     "@swc/core": "^1.2.162",
     "@swc/jest": "^0.2.20",
     "@types/jest": "^27.4.1",
     "@types/node": "18.11.7",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
     "auto": "^10.37.6",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-jsdoc": "^36.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
```

Comment:
Was this ever used? Asking as the latest eslint-config doesn't use it or include it.

Reply:
That's a very good point, and I am not sure when it was used - maybe in the beginning. But it looks to me like we don't need it anymore 🤔